examples/datastore/datastore-BigQuery.tf

```diff
@@ -0,0 +1,58 @@
+provider "satori" {
```

Comment:
The provider should not be part of the example, only a datastore...

Reply:
removed